clients/js/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.1",
   "dependencies": {
     "@celo-tools/celo-ethers-wrapper": "^0.1.0",
-    "@certusone/wormhole-sdk": "^0.8.0",
+    "@certusone/wormhole-sdk": "^0.9.2",
```

Comment:
Is this upgrade suitable for picking up the Injective package upgrades in a different PR?

Reply:
I believe so 🙂